universe/worlds/worlds.go

```diff
@@ -77,6 +79,79 @@ func (w *Worlds) GetWorlds() map[uuid.UUID]universe.World {
 	return worlds
 }
 
+func (w *Worlds) GetOptions(spaces map[uuid.UUID]universe.Space) ([]dto.ExploreOption, error) {
```

Comment:
seems like it's pure api method because it returns dto
in this case move it to api_worlds.go file and change name accordinally